text/0230-telemetry-viewer.md

```diff
@@ -0,0 +1,103 @@
+# Telemetry Viewer for Developers
+
+_A local telemetry viewer to aid in instrumentation and debugging of pipelines._
+
+## Motivation
+
+OpenTelemetry offers a rich, highly customizable, and highly configurable
+ecosystem of tooling, SDKs, APIs, and instrumentation libraries. However, with
+this complexity comes a cost -- the barrier to entry for new users can be very
+high, with significant cycle time required in order to understand how their
+instrumentation code changes affect the instrumentation emitted by their
+service.
+
+A 'local development' experience for OpenTelemetry would aid in reducing this
+cycle time and understandability burden from developers.
+
+## Explanation
+
+Different users of a telemetry system have different needs and expectations for
+debugging instrumentation. Currently, developers and coders have two options for
+quick feedback - using a logging exporter at the collector or SDK, or using an
+existing analysis tool (open source or proprietary). Both of these options have
+drawbacks - the logging exporter presents an overwhelming amount of text-based
+data, and depending the characteristics of a development environment, it may be
+challenging to stand up and use a local suite of open source analysis tools
+(such as Jaeger, Prometheus, and OpenSearch) or use a commercial tool.
+
```

Comment:
What comes on top of that is that those analysis tools lean towards use cases for large amounts of telemetry data and the capabilities to analyze them or to provide alerting/troubleshooting capabilities in an operational environment. A developer might have other needs, e.g. to watch the data in flux in chronological order or the list of libraries, agents, etc you listed below. Also, there is not a lot of assistance for troubleshooting your instrumentation (why do I not see traces? Why are my metrics having the wrong attributes? etc with very long cycles if you go via those tools.